modules/src/core/ics02_client/context.rs

```diff
@@ -78,6 +80,79 @@ pub trait ClientReader {
     fn client_counter(&self) -> Result<u64, Error>;
 }
 
+pub trait ConsensusReader {
```

Comment:
How about `LightClientReader`? And can we add a docstring that explins what this trait is for?

Reply:
https://github.com/informalsystems/ibc-rs/pull/2332/commits/f0ff0124ec6e06299b826c6c0add2ecbd332ac6b & https://github.com/informalsystems/ibc-rs/pull/2332/commits/a914b5b9e107e027eadf4a0da21651c3a3cf61a3